app/views/my_facilities/drug_stock/show.html.erb

```diff
@@ -0,0 +1,477 @@
+<div class="row">
+  <div class="col-lg-12">
+    <div class="card card-responsive">
+      <div>
+        <div class="d-flex align-baseline jc-between">
+          <h3>
+            Drug stock on hand: end of Dec-2020
+          </h3>
+          <p class="c-grey-dark">
+            Last updated: <%= @last_updated_at %>
+          </p>
+        </div>
+        
+        <p class="mb-4">
+          <!--a href="#" type="button" class="float-right ml-4 btn btn-sm btn-primary" data-toggle="modal" data-target="#DrugReportModal"><i class="fas fa-plus-circle mr-2"></i> Drug stock report</a-->
+          Patient days is calculated by comparing assigned patients against current stock on hand, normalized by estimated patients at each step of the hypertension protocol. Hover over cells to see calculation.
+        </p>
+        <p>
+          
+        </p>
+      </div>
+      <div class="d-flex fw-wrap mb-16px">
+        <%= render "shared/size_and_zone_filters" %>
+        <div class="dropdown show mb-8px mr-8px">
+          <a id="dropdownMenuLink" class="btn btn-sm btn-outline btn-secondary dropdown-toggle" href="#" role="button" data-toggle="dropdown" data-display="static" aria-haspopup="true" aria-expanded="false">
+              Dec-2020
+          </a>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+              <button class="dropdown-item active" value="" name="size" type="submit" form="query-filters" onclick="$('#selected-size').remove()">
+                Dec-2020
+              </button>
+              <button class="dropdown-item query-filter-size  filter-by-size" value="large" name="size" type="submit" form="query-filters" onclick="$('#selected-size').remove()">
+                Nov-2020
+              </button>
+              <button class="dropdown-item query-filter-size  filter-by-size" value="medium" name="size" type="submit" form="query-filters" onclick="$('#selected-size').remove()">
+                Oct-2020
+              </button>
+              <button class="dropdown-item query-filter-size  filter-by-size" value="small" name="size" type="submit" form="query-filters" onclick="$('#selected-size').remove()">
+                Sep-2020
+              </button>
+              <button class="dropdown-item query-filter-size  filter-by-size" value="community" name="size" type="submit" form="query-filters" onclick="$('#selected-size').remove()">
+                Aug-2020
+              </button>
+          </div>
+        </div>
+        <div class="d-flex fw-wrap mb-16px">
+          <button class="btn btn-sm btn-link dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
@dburka are you going to add more download options? If not, it makes more sense to make this a button and not a dropdown.

Also, this button is missing a border, is it intention that it doesn't have one?